### ISSUE TITLE
GitHub Channel

### ISSUE THREAD

**Issue by ManickYoj** [DakotaNelson/sneaky-creeper#63]
Seems like a good one to have.

**Comment by ManickYoj** [comment]
Seems like [github3](https://github.com/sigmavirus24/github3.py) or [PyGithub](https://github.com/PyGithub/PyGithub) are the most developed Github API libraries

**Comment by davinerd** [comment]
someone is already working on this? I can handle it.

**Comment by DakotaNelson** [comment]
Nope, I think this one is still open. Looking forward to seeing this in
action!
